data_sources/bro_conn.yml

```diff
@@ -0,0 +1,15 @@
+name: Bro conn
+id: c5a7e93b-2172-45a7-a7e9-3b217255a7f5
+version: 2
+date: '2025-01-23'
+author: Jacob Delgado, SnapAttack
+description: Logs network connection metadata captured by Zeek (formerly Bro), including
+  details such as source and destination IPs, ports, connection state, and protocol.
+mitre_components:
+- Network Connection Creation
+- Network Traffic Flow
+- Response Metadata
+- Application Log Content
+source: bro:conn:json
+sourcetype: bro:conn:json
+supported_TA: []
```

Comment:
Can we add this to supported TA  where applicable?

TA for Zeek
https://splunkbase.splunk.com/app/5466